apps/greenhouse/src/components/core-apps/org-admin/components/plugins/plugins/components/PluginList.tsx

```diff
@@ -30,11 +28,12 @@ const PluginList = () => {
 
   return (
     <>
-      <DataGridToolbar>
-        <ButtonRow>
+      <Stack className="mb-4">
```

Comment:
Please change this back to the DataGridToolbar to give the impression of one Toolbar block with buttons and filters.